perftracker/templates/regression_all.html

```diff
@@ -150,6 +150,32 @@
             });
         }
     });
+
```

Comment:
I'd add prefix as before the code above:

"3. Jobs to regressions linking"

Reply:
Don't get it